core-methods/src/main/java/com/mercadopago/sdk/android/coremethods/domain/interactor/CoreMethods.kt

```diff
@@ -201,8 +205,8 @@ class CoreMethods internal constructor(
             is Result.Success -> {
                 MPAnalytics.getInstance().trackMetric(
                     metricInstallmentsCallSuccess(
-                        paymentType = result.data.paymentTypeId.orEmpty(),
-                        merchantAccountId = result.data.merchantAccountId.orEmpty(),
+                        paymentType = result.data[0].paymentTypeId.orEmpty(),
+                        merchantAccountId = result.data[0].merchantAccountId.orEmpty(),
```

Comment:
This code could cause exceptions if the list is empty. Maybe we can swap for this
```suggestion
                        paymentType = result.data.getOrNull(0)?.paymentTypeId.orEmpty(),
                        merchantAccountId = result.data.getOrNull(0)?.merchantAccountId.orEmpty(),
```